homework3/cookie-clicker-hw2-5/src/main/webapp/WEB-INF/game.jsp

```diff
@@ -0,0 +1,184 @@
+            transform: translateX(20px);
+            color: ghostwhite;
+
+        }
+
+        .button-buy button:hover .tooltiptext {
+            visibility: visible;
+        }
+
+        .button-buy button:active{
+            transform: translateY(10px);
+            background-color: wheat;
+        }
+
+        .story{
+            background-color: transparent;
+            color: wheat;
+            overflow-y: scroll;
+            margin-right: 500px;
+            margin-left: 525px;
+            margin-bottom: 50px;
+            margin-top: 10px;
+
+        }
+
+        .title{
+            border: none;
+            margin: 20px;
+            margin-left: 450px;
+            color: wheat;
+            font-family: fantasy;
+            font-size: 30pt;
+
+
+
+        }
+
+
+        .jj{
+            display: flex;
+
+        }
+
+        .count{
+            margin-left: 450px;
+            font-size: 75pt;
+            font-family: fantasy;
+            color: wheat;
+            border: 0;
+            backface-visibility: hidden;
+        }
+
+        .clicker{
+            background-size: 500px 500px;
+            background-color: firebrick;
+            height: 500px;
+            width: 500px;
+            display: block;
+            margin-left: 250px;
+            background-repeat: no-repeat;
+            border: 0;
+        }
+
+        .clicker:active{
+            transform: translateY(5px);
+
+        }
+
+        .body{;
+            background-color: aqua;
+        }
+
+        .top-row{
+            display: flex;
+            justify-content: space-between;
+            font-family: cursive;
+        }
+
+        .tooltiptext {
+            visibility: hidden;
+            width: 320px;
+            height: 85px;
+            background-color: darkgray;
+            color: #fff;
+            text-align: center;
+            border-radius: 6px;
+            padding: 5px 0;
+
+            /* Position the tooltip */
+            position:absolute;
+            z-index: 1;
+        }
+
+        .cost{
+            display: flex;
+            justify-content: flex-end;
+            margin-bottom: 5px;
+            color: yellow;
+        }
+    </style>
+    <script>
+        generators = ${generator};
+        for(i = 0 ; i < generators.length ; i++){
+            generators[i].type = 'generators';
+            generators[i].unlockValue = generators[i].unlockAt;
+            generators[i].quantity = 0;
+            delete generators[i].unlockAt;
+            delete generators[i].id;
+        }
+        stories = ${stories};
+        for(i = 0 ; i < stories.length ; i++){
+            stories[i].triggeredAt = stories[i].triggerAt;
+            stories[i].state = 'hidden';
+            delete stories[i].triggerAt;
+            delete stories[i].id;
+        }
+        var backInfo = {};
+        backInfo.counter = 0;
+        backInfo.generators = generators;
+        backInfo.story = stories;
```

Comment:
`backInfo` is never used by the app.bundle.js (or from src/app.js before compile)